test/1_test_fullpath.js

```diff
@@ -1135,10 +1133,10 @@ contract('Voucher tests', async (addresses) => {
       buyerEscrowedAfter.gt(buyerEscrowedBefore);
 
       //Check VoucherKernel state
-      const voucherStatus = await contractVoucherKernel.vouchersStatus(
+      const voucherStatus = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey1
       );
-      assert.isTrue(voucherStatus.isPaymentReleased, 'Payment not released');
+      assert.isTrue(voucherStatus[1], 'Payment not released');
```

Comment:
Please use constants instead of hard-coded indexes for voucherStatus (same as remark above)